### ISSUE TITLE
(Anti)Social Media in 2020?

### ISSUE THREAD

**Issue by Ryochan7** [Ryochan7/DS4Windows#999]
I have been thinking about maybe branching out and making some social media accounts in 2020 specifically for my work on DS4Windows and other related gaming projects; I didn't really use my still active personal accounts in 2019 for much. That way people can get updates about DS4Windows without having to see some of the other stuff that I post online like Rocksmith videos.

**Comment by cristianmars22** [comment]
maybe get a comunity manager for social media , it would be really good, you have an amazing piece of work here defenetly deserves more visibility

**Comment by Tiberiusmoon** [comment]
Sounds good, may want to update the webpage for social media links and maybe go over the webpage to check the info is still valid. https://ryochan7.github.io/ds4windows-site/
maybe add the discord link too? (which i lost and cant seem to find again ^^;)

**Comment by Ryochan7** [comment]
I never got around to converting the site to use Jekyll or some other static site generator. Not sure when that will ever happen.

I have some general ideas but not any real plan. No community manager will be needed for the time being. This project is not that big and the only other project that might be under the umbrella for now is my version of Touchmote; maybe a future mapper as well if things go right in 2020. One big nope that I have set in stone is no Facebook. Some possible services that I would think about utilizing.

* Discord (existing)
* YouTube
* BitChute
* Twitter
* Mastodon (not on mastodon.social instance)
* Gab Social
* Minds (maybe)

Apart from that, the code for this project will probably be mirrored on a different site as well. I have an existing Bitbucket account that I seldomly use but not many people seem to be active on Bitbucket. GitLab seems to be more active. Becoming less reliant on GitHub would probably be a good idea for the future especially if Microsoft really messes up GitHub.

**Comment by Tiberiusmoon** [comment]
Discord and Twitter are good, Im surprised Facebook is still a thing and would probably die by the end of the year tbh.
YT is only good if you want to make video guides for each section of the app.
You could consider Imgur with a one off introduction GIF video to DS4.
The other things you mentioned ive never heard of but im not into social media.
Also how about Reddit? Game Devs use that a lot for user feedback

**Comment by Tiberiusmoon** [comment]
Also, if you wanna get more traffic you may wanna see if you can get your DS4 windows to be the first thing that shows up on google rather than third: https://i.imgur.com/0h6ttqr.jpg
Users could potentially blindly go for what sounds right and have no idea they are using Jays2kings old Builds.

**Comment by kurisu-gh** [comment]
> Also, if you wanna get more traffic you may wanna see if you can get your DS4 windows to be the first thing that shows up on google rather than third: https://i.imgur.com/0h6ttqr.jpg
> Users could potentially blindly go for what sounds right and have no idea they are using Jays2kings old Builds.

This. I was one of these people frustrated with DS4Windows and tried all sorts of other tools until I stumbled across [this Reddit post](https://www.reddit.com/r/pcgaming/comments/76ia26/psa_if_you_are_using_ds4windows_it_had_been/) about Ryochan7's repository. The version here is an amazing piece of work, props to @Ryochan7, @mika-n, and others who have contributed.

I occasionally visit the old Jay2Kings repo to scour for recent issues and redirect them to here. But I'm guessing a large amount of people just download the Jay2Kings version, come across a problem, and completely give up on it without attempting to go to the issues page.

**Comment by Ryochan7** [comment]
> I never got around to converting the site to use Jekyll or some other static site generator. Not sure when that will ever happen.

Spent the time last night to convert the site from static HTML pages to now being generated using Pelican SSG. What I mainly wanted was a minimal SSG with a templating language that allowed one site layout to be used for all generated pages. Pelican seems to fit the bill fairly well. Not sure if I would use the small blogging capabilities that are available but Pelican will be good enough for generating individual pages.

Hopefully maintaining the site will be easier in the future so I might actually do it more often. The site could probably use a different CSS template but I will wait on looking for a new template. The first order of business would obviously be to update the screenshots to reflect the current state of the program.

Pelican project site:
https://blog.getpelican.com/

**Comment by Ryochan7** [comment]
I will get some accounts set up today. I am going to forgo a more general brand for now and just set up project accounts for DS4Windows. Please don't handle snipe :smile:.

I have chosen to use Fosstodon as the main Mastodon instance. I have only recently gotten acquainted with Mastodon and taken a plunge into the Fediverse world in general. I have been interested in the general idea of federating content for a long time but I have never really liked any software that supports federation. That sentiment includes Mastodon and PeerTube which are two projects that I have been using regularly lately; they generally work well enough though. I feel like I have to make my own mark on the Fediverse landscape to better make it what I want it to be.

Other experiments, such as Touchmote, will still mainly be posted under personal accounts.

**Comment by Ryochan7** [comment]
Got the accounts set up. Need to make some content later. These links will be added to the README later.

Twitter: https://twitter.com/ds4windows
YouTube: https://www.youtube.com/channel/UCIoUA_XLlCSZbvZGeg3Byeg
BitChute: https://www.bitchute.com/channel/uE2CbiV96u1k/
Mastodon: https://fosstodon.org/@ds4windows
Gab Social: @ds4windows (link not included)
Minds: https://www.minds.com/ds4windows/

**Comment by Ryochan7** [comment]
Added GitLab link to previous post. The GitLab repository will just act as a mirror of this repository. Development of DS4Windows will continue to be done on GitHub.

https://gitlab.com/ryochan7/DS4Windows

**Comment by Ryochan7** [comment]
Although it is not social media, I will include a link to my backlog to help promote Backloggery.

https://backloggery.com/ryochan7

**Comment by Ryochan7** [comment]
Added LBRY and BitTube

LBRY: https://lbry.tv/@ds4windows
BitTube: https://bittube.tv/profile/ds4windows

**Comment by Rogus** [comment]
Hey @Ryochan7 can I get in touch with you, I have a serious problem with ds4v2 he was connecting just fine even though with those SCR or CSR I can't know for sure. But I bought one of these generic bluetooth dongles, but if I remember correctly it began with S not a C doesn't make any difference anymore, my pc almost got pretty destroyed by the heavy rains I live in the northeast part of Brazil so when it rains it rains pretty silently but this one was pretty heavy, it was a flood thank God and thank Jesus that my mother wake me up early I didn't even knew it was raining until she screamed to make long story short, my pc was wet down on water almost of it mainly in the usb entrances and in reset and power up and almost fricked out the whole thing out so I've tried to play with my micro usb cable from my phone I did stumbled by this image.
![Captura de tela 2020-03-02 11 07 31](https://user-images.githubusercontent.com/39033733/75684256-2e8bd100-5c77-11ea-8b72-25139085e156.png)
So sorry for the ramble it kinda got me scared of both losing the controller and the ability of playing with bluetooth wireless option and losing the way to play with micro usb cable btw the cable it is from a charger an original one btw from LG X POWER and in one part it says turbo for connecting at the charging entrance I did all those things like use dinput only installing xbox 360 drivers and vigem all again to repair to see if any kind of errors appear and my ds4 windows main windows shows shared mode and when I open steam it only recognizes the trackpad, that multitouch. Ah one more final question does ds4windows works for Denuvo protected games included with some kind of anticheat protection like EAC or BattlEye or any anticheat measure, so sorry again for the big post.

**Comment by Ryochan7** [comment]
I didn't realize that the old DS4Windows subreddit is still active. It doesn't matter to me though. I swore off of Reddit in 2016 when I deleted my old personal account. The site seems to have gotten even worse IMO and I don't want to go back.

**Comment by Ryusennin** [comment]
It's dead, Jim.

**Comment by Ryochan7** [comment]
What I expected: varying degrees of autism on display. Talking about games in general
What I got: bunch of people LARPing as Bill Lumbergh and sneak disses

The final straw on Discord was a person continuing to argue that DS4Windows does not need to exist because BetterJoy exists; at least that person didn't spam furry porn but maybe that would have happened later. People keep giving me reasons to disengage and just focus on my own ideas; Coreusa in the .NET 5 thread is my standard for what the general user base has been. On another note, getting spammed on Twitter for having the audacity to sleep should have been the time I started to ignore most people.

The Anti in Anti-Social was right. This whole experiment proved to be a massive failure. I am pretty much done even posting update announcements on Twitter.

Best Of Lumbergh
https://www.youtube.com/watch?v=kVmC0ktznNo
